www/doc/2.0/api-endpoints.html

```diff
@@ -227,5 +227,64 @@ <h2 id="mathjs">/api/mathjs</h2>
     <h2 id="localerror">/api/local-error</h2>
     <!--TODO--> Forthcoming.
 
+
+    <h2 id="cost">/api/cost</h2>
+
+    <details>
+    <summary>Example inputs & outputs</summary>
+
+    <p>Request:</p>
+
+    <pre>{
+        formula: &lt;FPCore expression&gt;,
+        sample: [point ... ]
+      }</pre>
+
+    <p>Response:</p>
+
+    <pre>{
+        cost: [cost]
+      }</pre>
+
+    <p>Specific Example: sqrt(x+1)-sqrt(x)</p>
+
+    <p>Request:</p>
+
+    <pre>{
+        formula: &lt;(FPCore (x) (- (sqrt (+ x 1)) (sqrt x)))&gt;,
+        sample: [ [[1], -1.4142135623730951] ]
+      }</pre>
+
+    <p>Response:</p>
+
+    <pre>{
+        cost: [13120]
+      }</pre>
+
+    <p>Specific Example Simplified: (x+1)-(x)</p>
+
+    <p>Request:</p>
+
+    <pre>{
+        formula: &lt;(FPCore (x) (- (+ x 1 ) x))&gt;,
+        sample: [ [[1], -1.4142135623730951] ]
+      }</pre>
+
+    <p>Response:</p>
+
+    <pre>{
+        cost: [320]
+      }</pre>
+    </details>
+
+    <p>The <code>cost</code> endpoint allows the user to request
+    the evaluation of an expression's overall cost. Cost is 
+    calculated depending on the complexity of operations contained
+    in the expression. </p>
+
```

Comment:
You should describe any arguments that would take a user by surprise. For example, we had questions about the `sample` argument when testing this endpoint, so you should note that it expects a sample of the kind generated by the /sample endpoint, like the other endpoints. You might also note that this sample is not currently used in the cost calculation, so it doesn't matter what it is as long as it's in the right format. But better practice would be for us to update the endpoint to use a default value when no value is passed. I will ask around about this issue.

Reply:
Done!